docs/changelogs/CHANGELOG-2.22.md

```diff
@@ -0,0 +1,481 @@
+- Add new field  `ReconciliationInterval` in `ApplicationInstallation` to force reconciliation, even if the `ApplicationInstallation` CR has not changed ([#11467](https://github.com/kubermatic/kubermatic/pull/11467))
+- Extend `ApplicationDefinition` and `ApplicationInstallation` CRD with `DeployOptions.HelmDeployOptions` to control how applications are deployed with `Helm`([#11608](https://github.com/kubermatic/kubermatic/pull/11608))
+    - ApplicationInstallation: set condition ready to `unknown` with reason `InstallationInProgress` before starting the installation
+    - ApplicationInstallation: don't try to install / upgrade the application if the max number of retries is exceeded 
+- Use string Version type for `ApplicationInstallation` CRD ([#11359](https://github.com/kubermatic/kubermatic/pull/11359))
+- Make uninstall for Applications idempotent ([#11622](https://github.com/kubermatic/kubermatic/pull/11622))
+- Add validating and defaulting webhook for Application deployOptions ([#11633](https://github.com/kubermatic/kubermatic/pull/11633))
+- Don't reuse values when upgrading Applications. The values defined in the ApplicationInstallation CR are the source of truth ([#11871](https://github.com/kubermatic/kubermatic/pull/11871))
+- Fix bug preventing deletion of an ApplicationInstallation if the ApplicationDefinition was removed before ([#11888](https://github.com/kubermatic/kubermatic/pull/11888))
+- Limit the number of retries only if deployOptions.Helm.atomic=true ([#11927](https://github.com/kubermatic/kubermatic/pull/11927))
+
+#### Konnectivity
+
+Konnectivity is now GA.
+
+- Set PriorityClassName of konnectivity-agent and openvpn-client to `system-cluster-critical` ([#11140](https://github.com/kubermatic/kubermatic/pull/11140))
+- Add keepalive-time Konnectivity setting + set keepalive to 1m by default ([#11502](https://github.com/kubermatic/kubermatic/pull/11502))
+- Remove `KonnectivityService` feature gate & make Konnectivity generally available ([#11643](https://github.com/kubermatic/kubermatic/pull/11643))
+- Update Konnectivity version to v0.0.35 ([#11657](https://github.com/kubermatic/kubermatic/pull/11657))
+
+#### Kubermatic Operator
+
+- The KKP operator can now perform the complete initial setup for new seed clusters. If MinIO/S3-exporter are not required, the KKP installer does not need to be used for setting up new / updating existing seed clusters ([#10795](https://github.com/kubermatic/kubermatic/pull/10795))
+
+#### Presets
+
+- Add `.spec.projects` field to `Preset` resources to allow binding Presets to specific projects ([#11100](https://github.com/kubermatic/kubermatic/pull/11100))
+
+#### OIDC
+
+- Add groups in OIDC kubeconfig ([#11121](https://github.com/kubermatic/kubermatic/pull/11121))
+- Add `OIDCProviderConfiguration` to Seed's spec allowing to configure dedicated OIDC provider for each Seed ([#11668](https://github.com/kubermatic/kubermatic/pull/11668))
+
+#### Resource Quotas (EE)
```

Comment:
We should also add the dyanmic quota feedback on the UI while creating a cluster

Reply:
Done.